### ISSUE TITLE
Your question took too long [using docker]

### ISSUE THREAD

**Issue by that0n3guy** [metabase/metabase#7587]
I've seen variants of this in other issues but I'm not finding a solution.

Metabase: v0.29.0
Install: official docker

I am running a query/question to postgress, takes 1-3-ish minutes using dbeaver.   Using metabase, it stops and gives me the 
```
Your question took too long
We didn't get an answer back from your database in time, so we had to stop. You can try again in a minute, or if the problem persists, you can email an admin to let them know.
```

How can I set it so that metabase (docker) will allow for longer query times?

**Comment by that0n3guy** [comment]
When I first loaded the card and after it timed out:
```
5/8/2018 3:06:18 PM05-08 15:06:18 DEBUG metabase.middleware :: GET /api/database 200 (217 ms) (9 DB calls). Jetty threads: 8/50 (4 busy, 4 idle, 0 queued)
5/8/2018 3:06:18 PM05-08 15:06:18 DEBUG metabase.middleware :: GET /api/card/167 200 (5 ms) (6 DB calls). Jetty threads: 8/50 (4 busy, 4 idle, 0 queued)
5/8/2018 3:06:18 PM05-08 15:06:18 DEBUG metabase.middleware :: GET /api/alert/question/167 200 (915 µs) (1 DB calls). Jetty threads: 8/50 (4 busy, 4 idle, 0 queued)
5/8/2018 3:06:55 PM05-08 15:06:55 ERROR generic-sql.query-processor :: Failed to set timezone:
5/8/2018 3:06:55 PM SQLException:
5/8/2018 3:06:55 PM Message: Connections could not be acquired from the underlying database!
5/8/2018 3:06:55 PM SQLState: null
5/8/2018 3:06:55 PM Error Code: 0
5/8/2018 3:06:55 PM
5/8/2018 3:06:55 PM05-08 15:06:55 ERROR generic-sql.query-processor :: Failed to set timezone:
5/8/2018 3:06:55 PM SQLException:
5/8/2018 3:06:55 PM Message: Connections could not be acquired from the underlying database!
5/8/2018 3:06:55 PM SQLState: null
5/8/2018 3:06:55 PM Error Code: 0
5/8/2018 3:06:55 PM
```

I'm not sure that is helpful.   If I click the "Get Answer" button now... I get nothing added to the logs.  

Just so you know, per https://www.metabase.com/docs/v0.19.2/operations-guide/running-metabase-on-docker.html I have JAVA_TIMEZONE set in my docker env vars, thought I don't know if that's the timezone its talking about.

A successful query to the same postgress db is: 
```
5/8/2018 3:11:19 PM05-08 15:11:19 DEBUG metabase.middleware :: POST /api/dataset 200 (136 ms) (15 DB calls). Jetty threads: 8/50 (5 busy, 3 idle, 0 queued)

```

**Comment by kyleabeauchamp** [comment]
I'd like to bump this issue as I think there are several dangling issues where folks would like to extend the timeout to allow a long-running query to complete, but in terms of dashboards and questions.

**Comment by flamber** [comment]
Closing this, since this was likely fixed in 0.32, and 0.35 introduced new connection handling.
Timeouts now only occur, when a proxy/load-balancer is closing the connection - #12423
